pkg/client/logging/dup_unix.go

```diff
@@ -21,3 +23,23 @@ func dupToStd(file *os.File) (err error) {
 	}
 	return err
 }
+
+func dupStd() (int, int, error) {
+	stdoutFd, err := unix.Dup(1)
+	if err != nil {
+		return -1, -1, err
+	}
+	stderrFd, err := unix.Dup(2)
+	if err != nil {
+		return -1, -1, err
+	}
+	return stdoutFd, stderrFd, err
+}
+
+func restoreStd(stdoutFd, stderrFd int) error {
+	err := unix.Dup2(stdoutFd, 1)
+	if err != nil {
+		return err
+	}
+	return unix.Dup2(stderrFd, 2)
+}
```

Comment:
Since these are only used by the tests, I'd prefer for them to be in `dup_{unix,windows}_test.go` files, rather than in the main `dup_unix.go` and `sysinfo_windows.go`.